lib/ansible/modules/crypto/openssl_certificate_info.py

```diff
@@ -238,10 +238,45 @@
                  or not.
     returned: success
     type: dict
+subject_key_identifier:
+    description:
+        - The CSR's subject key identifier.
+        - The identifier is returned in hexadecial, with C(:) used to separate bytes.
+        - Is C(none) if the C(SubjectKeyIdentifier) extension is not present.
+    returned: success and if the pyOpenSSL backend is I(not) used
+    type: str
+    sample: '00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff:00:11:22:33'
+    version_added: "2.9"
+authority_key_identifier:
+    description:
+        - The CSR's authority key identifier.
+        - The identifier is returned in hexadecial, with C(:) used to separate bytes.
+        - Is C(none) if the C(AuthorityKeyIdentifier) extension is not present.
+    returned: success and if the pyOpenSSL backend is I(not) used
+    type: str
+    sample: '00:11:22:33:44:55:66:77:88:99:aa:bb:cc:dd:ee:ff:00:11:22:33'
+    version_added: "2.9"
+authority_cert_issuer:
+    description:
+        - The CSR's authority cert issuer as a list of strings.
```

Comment:
```suggestion
        - The certificate issuer represented by C(authority_key_identifier).
```

I don't think a certificate can have multiple issuers? Unless it's a list of DN components, in which case it would make more sense to have "as a list of DN components"

Reply:
In https://tools.ietf.org/html/rfc5280#section-4.2.1.1 it is defined as `GeneralNames`. No idea why it is defined this way... How about `as a list of general names`?

Reply:
This was one where I tripped up again from being rusty on the RFC, that this isn't the same as the "issuer:" return field.